hogben/simulate.py

```diff
@@ -80,22 +102,19 @@ def simulate(sample, angle_times, scale=1, bkg=5e-6, dq=2, directbeam_path=None,
         scale (float): experimental scale factor.
         bkg (float): level of instrument background noise.
         dq (float): instrument resolution.
-        directbeam_path (str): path to directbeam file to simulate with.
+        instrument (str): path to directbeam file to simulate with.
```

Comment:
Same comment as above on the instrument description, and types no longer being needed in the docstring